supervision/config.py

```diff
@@ -1,2 +1,3 @@
 CLASS_NAME_DATA_FIELD = "class_name"
 ORIENTED_BOX_COORDINATES = "xyxyxyxy"
+TEXT_DATA_FIELD = "text_data"
```

Comment:
let's be more descriptive. I'm thinking: `OCR_TEXT_DATA_FIELD` and `"ocr_text"`

Reply:
@LinasKo  done